src/app/App.test.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Wait, the _tests_ need to be ".jsx"? That seems like an odd rule...

Reply:
Yeah. Any file with JSX in it needs a `.jsx` extension: